src/components/messages.js

```diff
@@ -730,6 +730,11 @@ const messages: { [string]: MessageDescriptor } = defineMessages({
         id: 'be.versionHistoryErrorHeaderMessage',
         description: 'Version history error message',
         defaultMessage: 'Something went wrong when fetching the version history.'
+    },
+    activityFeedTitle: {
```

Comment:
Since messages are coming from different/multiple files now, will have to change `copy-messages` in package.json to copy all messages from different files into their respective positions, so that `yarn link` doesn't break parent project for testing.

And since we have multiple files now, might as well move all activity feed stuff into activity feed folder.
And maybe sidebar stuff into sidebar folder.

Reply:
Maybe https://unix.stackexchange.com/questions/83593/copy-specific-file-type-keeping-the-folder-structure